src/deepsparse/server/config.py

```diff
@@ -120,6 +120,20 @@ class ServerConfig(BaseModel):
         description="The kind of integration to use. local|sagemaker",
     )
 
+    engine_thread_pinning: bool = Field(
```

Comment:
let's actually match here (see logic in the same file): https://github.com/neuralmagic/deepsparse/blob/87b2b5d900ac8de1ad348be72fc30f739b06c545/src/deepsparse/benchmark/benchmark_model.py#L203

forgot there's also the numa option